decidim-regulations/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb

```diff
@@ -5,17 +5,18 @@ module ParticipatoryProcesses
     # A controller that holds the logic to show ParticipatoryProcesses in a
     # public layout.
     class ParticipatoryProcessesController < Decidim::ApplicationController
-      layout "layouts/decidim/participatory_process", only: [:show]
-
-      before_action -> { extend NeedsParticipatoryProcess }, only: [:show]
+      include ParticipatorySpaceContext
+      participatory_space_layout only: :show
 
       helper Decidim::AttachmentsHelper
       helper Decidim::IconHelper
       helper Decidim::WidgetUrlsHelper
+      helper Decidim::SanitizeHelper
```

Comment:
Why this SanitizerHelper all around?

Reply:
because new features have been added to version 8 and these controllers are overwritten